content/projects/understanding-loops/_index.md

```diff
@@ -65,6 +65,8 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ## Write some functionality
 
+When writing your functionality, please be aware that the parameters should not be limited to a certain order. For example, `getCustomerBaskets(someJsonData, "sine@umuzi.org")` and `getCustomerBaskets("sine@umuzi.org", someJsonData)` should work the same way and produce the same results. Using destructuring helps to achieve this. It is important, however, to note that when using destructuring, the names of the arguments will matter a lot as they have to match wherever they are being used in the code. Here is more on [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
```

Comment:
Golly. 

This means that inside the function I would need to check what argument is a string resembling an email, and which one is an object. 

This is not a good instruction to follow. It will confuse pretty much everyone. 

Also destructuring is only relevant for javascript people, not for Python or Java people. So we need to make sure that only the Javascript people are directed to do restructuring.

Maybe say

## Important notes for Javascript solutions

In javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: Link

Please make sure you use restructuring for all your functions. 

```
getCustomerBaskets(someJsonData, "sine@umuzi.org") // DONT DO THIS
getCustomerBaskets("sine@umuzi.org", someJsonData) // DONT DO THIS EITHER

// the problem with the above code is that if you get the order of the parameters wrong, then things break. 
// your functions should work like this instead:

// what should it look like?? Put in the real thing. Not a wrong thing
```